docs/hello-world.md

```diff
@@ -0,0 +1,124 @@
+## Hello World
```

Comment:
(or "getting-started-with-node-and-npm.md" if we want something more descriptive and SEO friendly